app/services/aga_td_list.rb

```diff
@@ -61,6 +61,8 @@ def self.parse(ids = nil)
     Rails.cache.fetch(CACHE_KEY, :expires_in => 24.hours) do
       parsed_aga_member_info = {}
       tsv = fetch()
+      puts "tsv?"
+      puts tsv
```

Comment:
Super classy to have accidentally committed this, then had to remove it on the other commit in this PR. 🙄